script.sh

```diff
@@ -0,0 +1,2 @@
+sudo chmod +x /usr/local/bin/docker-compose
```

Comment:
da li si testirala ovo? ne vidim nigde instalaciju docker-a i docker-compose-a. Ako je to pokrenuto na neki drugi nacin, linkuj mi molim te taj deo :)

Reply:
Moja skripta je na pocetku izgledala ovako:
curl -fsSL https://get.docker.com -o get-docker.sh
sudo sh get-docker.sh
sudo curl -L "https://github.com/docker/compose/releases/download/1.27.4/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
sudo chmod +x /usr/local/bin/docker-compose
docker-compose up --build 

Medjutim na pocetku pokretanja mi je izlazilo neko obavestenje da vec imam instaliran docker ili docker skriptu, tako nesto, i da to moze izazvati probleme. I na taj nacin mi se pokrene i aplikacija i baza ali bazi ne mogu da pristupim kao da nije pokrenuta, a ni aplikaciji. Tek onda kada sam obrisala ove linije koda mi je proradilo normlano. Mogu vratiti ove linije za instaliciju pa da vam posaljem upozorenje koje mi je iskakalo, ja stvarno ne znam sta je bio problem :D

Reply:
trebalo bi d je samo obavestenje da postoje instalirani paketi i da svakako radi kako treba, ali vrati pa mi posalji gresku. Ovako ce raditi sada kod tebe, ali je problem sto onda infrastruktura nije ponovljiva i ako recimo ja pokusam ovo da pokrenem kod sebe, nece mi instalirati docker i docker-compose

Reply:
![dockerExist](https://user-images.githubusercontent.com/57102452/103312053-37c30700-4a1c-11eb-84d0-a2c27b6ced70.png)

Reply:
Sada mi radi kada ovde ukucam Ctrl+C, ali identicno ovo sam juce probavala i nije radilo. Nadam se da je sve okej sada :)